test/controllers/episodes_controller_test.rb

```diff
@@ -59,8 +53,11 @@ class EpisodesControllerTest < ActionDispatch::IntegrationTest
   end
 
   test "should update episode" do
-    patch episode_url(episode), params: {episode: params}
-    assert_redirected_to edit_episode_url(episode)
+    Episode.stub_any_instance(:copy_media, true) do
```

Comment:
Where does `:copy_media` come from?

Reply:
A bunch of our models (Podcast, Episode, Feed, Images, MediaResources) have a `def copy_media` method on them.  Which triggers Porter to inspect/copy S3 files to their destinations.

I had to mock it out in the test now - since I added `@episode.copy_media` to the controller.  Otherwise the test suite complains that some code made an un-expected outgoing HTTP request during the test run.